### ISSUE TITLE
setting resource limits causes bad behaviour for unprivileged containers

### ISSUE THREAD

**Issue by xmcqueen** [hashicorp/nomad-driver-lxc#11]
I've started working on this. It looks to be not very active.  I'm wondering about the state of this plugin.  I'm working on getting it going for unprivileged containers.  Its working at this point, but I want to take it further.

I noticed when I use the existing plugin's setResourceLimits, I get some odd behavior. One of which is when I add resource limits, the plugin starts the unprivileged containers, and then starts a whole new set of privileged one.

I'm looking for warnings or advice from any maintainers, since I'm about to push ahead and make this work.

- Is this plugin designed to run as root and do only privileged containers?
- If so should I try to make it do both privileged and unprivileged or should I just go rogue

I've got some related questions about the design assumptions, so I guess if there are any maintainers I'd like to chat.

**Comment by towe75** [comment]
@xmcqueen i'd like to contribute also some stuff to this project and filed 2 pull requests. Maybe we can work somehow together? 
Please check my proposal/question in #14 regarding a better build environment. What do you think about it?

**Comment by xmcqueen** [comment]
Yes, I'm interested.  I'll check it out when I catch up.  I've been out-of-town for some weeks.  I'm glad to hear from somebody else.
